src/live/handleAlbumChat.js

```diff
@@ -0,0 +1,14 @@
+function handleAlbumChat(io, socket, data) {
+  const { message, userId, albumId } = data;
+
+  // Create a chat room name for the album chat
+  const chatRoomName = `chat:album:${albumId}`;
+
+  // Join the album chat room
+  socket.join(chatRoomName);
```

Comment:
The socket _should_ already have joined this room (by sending a \`join:album\_chat\` message earlier) to be sending a chat to it now; so we probably don't need this line. But it's probably fine to leave for now as a way to ensure they're in the room I suppose :thinking:

Reply:
Removed

Reply:
<img width="682" alt="postman" src="https://github.com/Code-the-Dream-School/dd-prac-team4-back/assets/95896159/97ddc618-7e2c-48f5-a3ce-e4ebaf1f56c7">

Reply:
@akosasante Look like its work

Reply:
@akosasante Thank you so much for such a detailed and clear explanation. I'd love to take any programming course from you. You explain things so wonderfully!   

I tried as you explained, and everything works. If possible, it would be good to leave the console.log for further exploration, and the girls can also watch the video and test it themselves.